components/entitlement/org.wso2.carbon.identity.entitlement/src/main/java/org/wso2/carbon/identity/entitlement/internal/EntitlementExtensionBuilder.java

```diff
@@ -374,12 +374,12 @@ private void populatePolicyCollection(Properties properties, EntitlementConfigHo
     private void populatePolicyStoreModule(Properties properties, EntitlementConfigHolder holder)
             throws Exception {
 
-        PolicyStoreManageModule policyStoreStore = null;
+        PolicyDAO policyStoreStore = null;
 
         if (properties.getProperty("PDP.Policy.Store.Module") != null) {
             String className = properties.getProperty("PDP.Policy.Store.Module");
             Class clazz = Thread.currentThread().getContextClassLoader().loadClass(className);
-            policyStoreStore = (PolicyStoreManageModule) clazz.newInstance();
```

Comment:
'PDP.Policy.Store.Module' type used to be `PolicyStoreManageModule` but with this change its needs to be the type `PolicyDAO`.